### PR TITLE
Add some additional functions to the KeySpace/TableChanger interface

### PR DESCRIPTION
**Pull request by obeattie**
- `Tables()` and `Exists()` were already present on the underlying `table` object, but were not exposed
- `Name()` returns the underlying C\* name of a keyspace
- `QueryExecutor()` returns the configured `QueryExecutor` for the keyspace

Oh, and allow specification of a `GOCASSA_TEST_HOSTS` environment variable, to control which hosts+ports are used during testing.

### CODE REVIEW

**Comment by crufter** [comment]
Wonder what would we break by renaming Tables to TableNames.

File: interfaces.go [hunk 1]
```diff
@@ -128,6 +134,8 @@ type TableChanger interface {
 	Create() error
 	CreateStatement() (string, error)
 	Recreate() error
+	// Name returns the name of the keyspace, as in C*
+	Name() string
```
Comment (crufter) [inline interfaces.go:157]: I don't get this. Seems like you implemented Name for `KeySpace` below, not for TableChanger.
Returning `KeySpace` name from tables is a slippery slope anyway, I would rather return the `KeySpace` type a table belongs to, or, I would rather not return any of this tbh.

Comment (obeattie) [inline interfaces.go:157]: The comment is wrong. It returns the keyspace name. I'll change it.

**Comment by crufter** [comment]
Can we not include `QueryExecutor` yet unless it is already sorely needed?

My biggest problem with this PR was that I am trying to decouple gocassa from CQL as much as possible and every CQL specific method (like the already existing CreateStatement) rubs me in a wrong way. I like how at the moment is hidden from the interface (although it is passed in/implied when constructing the `KeySpace`, but those are different things).

As a compromise, can have something like the `TableChanger` interface which hides these somewhat from the user? Something like

``` go
// KeySpace is used to obtain tables from.
type KeySpace interface {
    MapTable(tableName, id string, row interface{}) MapTable
    MultimapTable(tableName, fieldToIndexBy, uniqueKey string, row interface{}) MultimapTable
    TimeSeriesTable(tableName, timeField, uniqueKey string, bucketSize time.Duration, row interface{}) TimeSeriesTable
    MultiTimeSeriesTable(tableName, fieldToIndexByField, timeField, uniqueKey string, bucketSize time.Duration, row interface{}) MultiTimeSeriesTable
    Table(tableName string, row interface{}, keys Keys) Table
    SomethingSomething
}

type SomethingSomething interface {
    // Name returns the keyspace name as in C*
    Name() string
    // QueryExecutor returns the configured executor for this KeySpace
    QueryExecutor() QueryExecutor
    // Tables returns the name of all configured column families in this keyspace
    Tables() ([]string, error)
    // Exists returns whether the specified column family exists within the keyspace
    Exists(string) (bool, error)
}
```

Just throwing ideas around.

**Pull request merged**